src/dmd/dsymbol.d

```diff
@@ -1042,51 +1042,54 @@ extern (C++) class Dsymbol : ASTNode
     {
         //printf("Dsymbol::oneMembers() %d\n", members ? members.dim : 0);
         Dsymbol s = null;
-        if (members)
+        if (!members)
         {
-            for (size_t i = 0; i < members.dim; i++)
+            *ps = null;
```

Comment:
Adding this line changes the behavior, making this change not trivial and not a clear refactor.